lib/sky-pages-module-generator.js

```diff
@@ -244,6 +249,12 @@ export function skyViewkeeperHostOptionsFactory(config: SkyAppConfig): SkyViewke
   return hostOptions;
 }
 
+export function paramsConfigFactory(config: SkyAppConfig): SkyAppParamsConfig {
```

Comment:
Should we call this `skyAppParamsConfigFactory` since we're exporting it?

Reply:
Done. Regarding your top comment, the `SkyAppRuntimeConfigParamsProvider` is provided in the `'root'`, so consumers can pull it into their components/services without a module providing it (or Builder, in this case).
https://github.com/blackbaud/skyux-config/blob/master/src/app/public/params-provider.ts#L17

Reply:
Got it.